src/liboslexec/llvm_instance.cpp

```diff
@@ -2225,7 +2225,7 @@ BackendLLVM::run()
             // The triple is empty with recent versions of LLVM (e.g., 15) for reasons that aren't
             // clear. So we must set them to the expected values.
             // See: https://llvm.org/docs/NVPTXUsage.html
-            ll.module()->setTargetTriple("nvptx64-nvidia-cuda");
+            ll.module()->setTargetTriple(llvm::Triple("nvptx64-nvidia-cuda"));
```

Comment:
This one doesn't have a `#if`. Do you expect it to work across all of the LLVM versions we claim to support?